Tests/Tests.pro

```diff
@@ -2,7 +2,7 @@ TEMPLATE = app
 QT += testlib
 CONFIG += testcase
 
-LIBS += -L../../build/.lib -LC:\gtest\lib -lViewLayer -lBusinessLayer -lCommunicationLayer -lPresenterLayer -lDataLayer -lgmock -lgtest
+LIBS += -L../../build/.lib -LC:\gtest\lib -lViewLayer -lBusinessLayer -lCommunicationLayer -lPresenterLayer -lDataLayer libgmock.a
```

Comment:
Change this line to UNIX only. Remove -LC:\gtest\lib.

For now on windows you could see if you can just disable the test building unless you can get it working.